ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/TestWorker.java

```diff
@@ -17,7 +17,6 @@
  */
 package org.apache.hadoop.hive.ql.txn.compactor;
 
-import it.unimi.dsi.fastutil.booleans.AbstractBooleanBidirectionalIterator;
```

Comment:
is this related?

Reply:
Yes, this need `it.unimi.dsi:fastutil`, we have removed this dependency:
![image](https://user-images.githubusercontent.com/5399861/100756200-f6910480-3427-11eb-9919-af782b870b9e.png)

Reply:
Gotcha, cool.